delphin/codecs/dmrx.py

```diff
@@ -295,7 +295,10 @@ def _encode_node(node, properties, lnk):
         attributes['carg'] = node.carg
     e = etree.Element('node', attrib=attributes)
     e.append(_encode_pred(node.predicate))
-    e.append(etree.Element('sortinfo', attrib=node.sortinfo))
+    if properties is True:
+        e.append(etree.Element('sortinfo', attrib=node.sortinfo))
+    else:
+        e.append(etree.Element('sortinfo'))
```

Comment:
If we only care about the truth value (and not that the value is exactly the singleton `True`) then the preferred form is:

```python
if properties:
    ...
```

If we want to reduce some redundancy in the code, we can also do an inline conditional:

```python
e.append(etree.Element('sortinfo',
                       attrib=node.sortinfo if properties else {}))
```